docker/cellranger/7.0.1/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:bullseye-slim
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y alien unzip build-essential dpkg-dev curl gnupg procps python3 python3-pip
```

Comment:
Please install `rsync` as well, as this is not included in Debian images by default, but user may need it for `local` backend.